controllers/component_build_controller_test.go

```diff
@@ -1288,6 +1409,12 @@ var _ = Describe("Component initial build controller", func() {
 			ensureNoPipelineRunsCreated(resourceKey)
 		})
 
+		It("should not submit initial build if pac secret is missing", func() {
+			deleteSecret(pacSecretKey)
+			setComponentDevfileModel(resourceKey)
+			ensureNoPipelineRunsCreated(resourceKey)
```

Comment:
I'd rather check status annotation instead of using `ensureNoPipelineRunsCreated`, where possible.

Reply:
ok will add